app-rails/config/locales/defaults/en.yml

```
@@ -46,6 +46,10 @@ en:
       default: "%m/%d/%Y"
       short: "%b %d"
       long: "%B %d, %Y"
+  errors:
+    attributes:
+      hp_field:
+        present: "You entered text in a field that is meant to catch spambots."
```

Perhaps something like this to make it clearer what the user should do to be able to pass validation?
```suggestion
        present: "This field should be left empty. It is intended to prevent bots from submitting spam using this form."
```